src/kweb/Makefile

```diff
@@ -1,2 +1,2 @@
 run:
-	uvicorn main:app --reload
+	uvicorn main:app --reload --host 0.0.0.0
```

Comment:
With this you are exposing kweb to the whole network. I would advise strongly against this unless people know what theu are doing. kweb is in no way hardened against any attacks and should never do this unless specifically running it in a docker env similar.

Esp. on universities you have often public IPs assigned to even work stations. Usually they have firewalls in place. But still, don't do this in a make filw.

Reply:
Aye, inside the container it is however necessary for the host system to be able to talk to it

Reply:
Probably would be best to use docker-compose for this.

Reply:
Maybe we can add a second target for running inside docker?

Reply:
https://fastapi.tiangolo.com/deployment/docker/ also suggests to use `--host 0.0.0.0` for within docker container.

Reply:
Yes, I never disputed of doing it in docker. But you are adding it to the make file as a default.  There a multiple options to add it so that it is not just a hidden arg in the Makefile:

Options:
  * use a env variable that is set in docker, e.g. something like `ARG HOST="--host 0.0.0.0"`
  * don't use make and directly call the entrypoint with the full command
  * use docker compose

Reply:
@sebastian-goeldi, I have removed the `--host` flag in the Makefile and only use it in the entrypoint now, as it is also done in the link shared above. Is that more acceptable?

Reply:
Yes, perfect